src/main/java/com/cinema/model/Movie.java

```diff
@@ -0,0 +1,46 @@
+package com.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
```

Comment:
it is not the best practice to use the table name with UpperCase first letter. Try to add annotation table after entity with parameter 'name'